vllm/engine/llm_engine.py

```diff
@@ -140,7 +142,10 @@ def get_tokenizer_for_seq(self, sequence: Sequence):
     def _init_workers(self):
         # Lazy import the Worker to avoid importing torch.cuda/xformers
         # before CUDA_VISIBLE_DEVICES is set in the Worker
-        from vllm.worker.worker import Worker
+        if self.is_neuron:
+            from vllm.worker.neuron_worker import Worker
+        else:
+            from vllm.worker.worker import Worker
```

Comment:
Can we change this to a dictionary from hardware to workers?

Reply:
changed to a dict to load the modules dynamically based on device config.